Src/Campus.Services.Interfaces/DTO/User/UserRegistrationDto.cs

```diff
@@ -0,0 +1,12 @@
+namespace Campus.Services.Interfaces.DTO.User
+{
+    public class UserRegistrationDto
+    {
+        public string Email { get; set; }
+        public string UserName { get; set; }
```

Comment:
Is UserName field intended for holding Nickname, If so, consider changing to Nickname

Reply:
The `UserName` is default field in the `Core Identity`